include/http2.hrl

```diff
@@ -85,8 +85,8 @@
 
 -type stream_id() :: non_neg_integer().
 -record(frame_header, {
-    length      :: non_neg_integer(),
-    type        :: frame_type(),
+    length      :: non_neg_integer() | undefined,
```

Comment:
@joedevivo the issue is this code does make length undefined, https://github.com/joedevivo/chatterbox/blob/master/src/h2_stream.erl#L554-L556